### ISSUE TITLE
Claims "MalformedYAML" but passes yamllint.

### ISSUE THREAD

**Issue by pillarsdotnet** [mattrobenolt/jinja2-cli#58]
I'm trying to debug a jinja2 template.  The associated variables file passes yamllint, but jinja-cli claims that it is malformed.  Any suggests as to how I can proceed?

```
user:~$ sudo -i bash -c 'umask 002 ; pip install --force-reinstall -U yamllint jinja2-cli'
Collecting yamllint
  Using cached https://files.pythonhosted.org/packages/87/ca/9866f1c49301efb1f9c3b03eb879a95611e71dd56f73d2d45d537a5e1518/yamllint-1.11.1-py2.py3-none-any.whl
Collecting jinja2-cli
  Using cached https://files.pythonhosted.org/packages/ca/0f/dd461ac244fec4310c425e9cec9553d8ae8523c36837792e7666c6e53393/jinja2_cli-0.6.0-py2.py3-none-any.whl
Collecting pathspec>=0.5.3 (from yamllint)
Collecting pyyaml (from yamllint)
Collecting jinja2 (from jinja2-cli)
  Using cached https://files.pythonhosted.org/packages/7f/ff/ae64bacdfc95f27a016a7bed8e8686763ba4d277a78ca76f32659220a731/Jinja2-2.10-py2.py3-none-any.whl
Collecting MarkupSafe>=0.23 (from jinja2->jinja2-cli)
Installing collected packages: pathspec, pyyaml, yamllint, MarkupSafe, jinja2, jinja2-cli
  Found existing installation: pathspec 0.5.7
    Uninstalling pathspec-0.5.7:
      Successfully uninstalled pathspec-0.5.7
  Found existing installation: PyYAML 3.13
    Uninstalling PyYAML-3.13:
      Successfully uninstalled PyYAML-3.13
  Found existing installation: yamllint 1.11.1
    Uninstalling yamllint-1.11.1:
      Successfully uninstalled yamllint-1.11.1
  Found existing installation: MarkupSafe 1.0
    Uninstalling MarkupSafe-1.0:
      Successfully uninstalled MarkupSafe-1.0
  Found existing installation: Jinja2 2.10
    Uninstalling Jinja2-2.10:
      Successfully uninstalled Jinja2-2.10
  Found existing installation: jinja2-cli 0.6.0
    Uninstalling jinja2-cli-0.6.0:
      Successfully uninstalled jinja2-cli-0.6.0
Successfully installed MarkupSafe-1.0 jinja2-2.10 jinja2-cli-0.6.0 pathspec-0.5.7 pyyaml-3.13 yamllint-1.11.1
user:~$ yamllint test.yml
user:~$ jinja2  --format=yml test.j2 test.yml
Traceback (most recent call last):
  File "/usr/local/bin/jinja2", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/jinja2cli/cli.py", line 335, in main
    sys.exit(cli(opts, args))
  File "/usr/local/lib/python2.7/dist-packages/jinja2cli/cli.py", line 234, in cli
    raise raise_exc(u'%s ...' % data[:60])
jinja2cli.cli.MalformedYAML: ---

wazuh_agent_configs:
  - .os: 'Linux'
    localfile:
   ...
user:~$
```

**Comment by mattrobenolt** [comment]
Can you share the full yaml file? Or some anonymized version that can cause a failure?

It seems maybe like you're trying to use a multi-document yaml file. Not sure exactly what they're called off the top of my head, but just going by the `jinja2cli.cli.MalformedYAML: ---` error, that seems possible?

**Comment by pillarsdotnet** [comment]
That is certainly possible, as I generated the file by concatenating several other files.

I've since switched to calling jinja directly from python.  I've tried three different jinja2 cli programs, including yours, and none of them will tell me the line number on which an error occurred.  But when invoking jinja2 directly from python, an error backtrace *does* include this vital bit of debugging information.
